package.json

```diff
@@ -1,23 +1,26 @@
 {
-  "name": "chat_app_api",
-  "description": "Backend Service For The Chat APP",
-  "version": "1.0.0",
-  "main": "index.js",
-  "license": "MIT",
-  "scripts": {
-    "dev": "nodemon src/index.ts"
-  },
-  "devDependencies": {
-    "@types/cors": "^2.8.13",
-    "@types/express": "^4.17.17",
-    "@types/node": "^18.14.6",
-    "nodemon": "^2.0.21",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.9.5"
-  },
-  "dependencies": {
-    "cors": "^2.8.5",
-    "dotenv": "^16.0.3",
-    "express": "^4.18.2"
-  }
+	"name": "chat_app_api",
+	"description": "Backend Service For The Chat APP",
+	"version": "1.0.0",
+	"main": "index.js",
+	"license": "MIT",
+	"scripts": {
+		"dev": "nodemon src/index.ts",
+		"build": "rimraf /dist && tsc",
+		"start": "yarn run build && node dist/index.js"
+	},
+	"devDependencies": {
+		"@types/cors": "^2.8.13",
+		"@types/express": "^4.17.17",
+		"@types/node": "^18.14.6",
+		"nodemon": "^2.0.21",
+		"rimraf": "^4.4.0",
```

Comment:
Why this package? you can achieve the same thing with the standard linux command `rm -rf`.

Reply:
that is exactly what the `rimraf` is doing, deleting the built files when we run yarn start before building again, to avoid having old built files along with the new ones when changes are being made

Reply:
ooh, I think I get it now, instead of installing the package, using `rm -rf` in the script does exactly the same thing, tested that.